Makefile

```diff
@@ -1,9 +0,0 @@
-all:
-	composer install --no-dev
-	npm run compile
-	git describe --tags --always > VERSION
-
-dev:
```

Comment:
This functionality is preserved by `setup` and `bootstrap`; one can add a 'dev' argument upon invoking the script and have composer do the appropriate thing.